js/cookies.js

```diff
@@ -1,28 +1,63 @@
+// The duration of each cookie
+const duration = 20;
+
+// Save a cookie with indicated duration
 function saveCookie(firstName, lastName, ID)
 {
-    const duration = 20;
 	let date = new Date();
 	date.setTime(date.getTime()+(duration*60*1000));	
 	document.cookie = JSON.stringify({
-        "firstName" : firstName,
-        "lastName" : lastName,
+        "FirstName" : firstName,
+        "LastName" : lastName,
         "ID" : ID,
-        "expires:" : date.toGMTString()
+        "Expires" : date.toGMTString()
```

Comment:
I think this can just be changed to:

```js
document.cookie = "";
```

It's going to get overwritten in `saveCookie()` anyways.

Reply:
Sorry this comment should be in `clearCookie` not `saveCookie`

Reply:
Fair, updated.